frappe/public/js/frappe/ui/toolbar/awesome_bar.js

```diff
@@ -84,15 +84,15 @@ frappe.search = {
 			onclick: function() {
 				var txt = '<table class="table table-bordered">\
 					<tr><td style="width: 50%">'+__("Make a new record")+'</td><td>'+
-						__("<b>new</b> <i>type of document</i>")+'</td></tr>\
+						__("new type of document")+'</td></tr>\
```

Comment:
In such cases, we can use markdown, as we need the bold and emphasis
@rmehta what do you think?